include/libp2p/muxer/yamux/yamuxed_connection.hpp

```diff
@@ -165,15 +168,31 @@ namespace libp2p::connection {
     void onDataWritten(outcome::result<size_t> res, StreamId stream_id,
                        bool some);
 
+    /// Creates new yamux stream
+    std::shared_ptr<Stream> createStream(StreamId stream_id);
+
     /// Erases stream
     void eraseStream(StreamId stream_id);
 
+    /// Erases stream
```

Comment:
Stream or streams? Not clear from the description how the iterator is going to be used.